### ISSUE TITLE
Raiton/Select abilities registering 0 Damage incorrectly.

### ISSUE THREAD

**Issue by Lunaretic** [ravahn/FFXIV_ACT_Plugin#40]
Some abilities seem to be registering as 0/NaN damage incorrectly.  Raiton being a simple go-to example.  Some information is available in this reddit thread: https://www.reddit.com/r/ffxiv/comments/3ghr0g/act_currently_parsing_incorrect_numbers/

Issue persists through both DX9 and DX11, with network parsing enabled or disabled.

Additionally, logs from the plugins' log directory for Raiton (DX11 client, network parsing enabled, ACT v3.2.0, Plugin v1.4.1.7): 

05|2015-08-10T14:30:59.3660000-04:00|16d|0||105d0819|Kalina Skysong|30.00|a|5659
05|2015-08-10T14:30:59.3660000-04:00|16d|0||1057bb9f|Emmaleagh Kapelski|30.00|a|84
21|2015-08-10T14:31:00.2030000-04:00|105D0819||8D3|Ten|105D0819||0|0|0|0|0|0|0|0|1000011|1F0|0|0|0|0|0|0|
00|2015-08-10T14:30:59.0000000-04:00|082b||You use Ten.
00|2015-08-10T14:30:59.0000000-04:00|08ae|| You make the gesture for “heaven.”
26|2015-08-10T14:31:00.4050000-04:00|1f0|Mudra|5.00|105D0819||105D0819||01|5659|5659|
05|2015-08-10T14:31:00.0920000-04:00|1f0|105d0819|Kalina Skysong|105d0819|Kalina Skysong|4.93|1|5659
21|2015-08-10T14:31:00.7820000-04:00|105D0819|Kalina Skysong|8D5|Chi|105D0819|Kalina Skysong|0|0|0|0|0|0|0|0|15000011|1F0|0|0|0|0|0|0|5659|5659|1274|1274|1000|1000|244.7604|-219.4694|111.8341|5659|5659|1274|1274|1000|1000|244.7604|-219.4694|111.8341
00|2015-08-10T14:31:00.0000000-04:00|082b||You use Chi.
00|2015-08-10T14:31:00.0000000-04:00|08ae|| You make the gesture for “earth.”
21|2015-08-10T14:31:02.4620000-04:00|105D0819|Kalina Skysong|8DB|Raiton|4000022A|Striking Dummy|0|0|0|0|0|0|0|0|50503|28A|0|0|0|0|0|0|44|44|0|0|1000|1000|253.5589|-214.0755|105.9843|5659|5659|1274|1274|1000|1000|244.7604|-219.4694|111.8341
00|2015-08-10T14:31:01.0000000-04:00|08b0||You lose the effect of Mudra.
06|2015-08-10T14:31:01.9720000-04:00|1f0|105d0819|Kalina Skysong|105d0819|Kalina Skysong|3.58|15|5659
00|2015-08-10T14:31:02.0000000-04:00|082b||You use Raiton.
00|2015-08-10T14:31:02.0000000-04:00|0aa9||   The striking dummy takes 650 damage.

Possibly related or unrelated, also seeing some issues with Mutilate reporting inaccurate numbers and stopping increasing in damage/stopping ticking after the first ~4 ticks/12 seconds (30 second DoT).  Please let me know if I should submit information on that as a new ticket or if it might be related to this.

EDIT: Primary Issue solved, realized I had 'Parse Potency Numbers' enabled, which was causing confusion.  That noted, the potency for Raiton seems to be calculated as 0 by ACT.

**Comment by Lunaretic** [comment]
As noted in edit, primary issue was caused by the 'Parse Potency Numbers' option.  That said, the potency number on Raiton seems to be calculated as 0, so there is still a bug to be fixed there (ergo not closing this issue myself).

**Comment by ravahn** [comment]
I removed the potency values for nin magic-based skills because during patch 3.0 testing I found some evidence that they were not producing consistent numbers when compared to physical hits.  However, this needs more investigation - it kind of slipped through the cracks, but mostly affects the 'Parse Potency Numbers' feature.

I'll make a note to dig into it in more detail.  It may have been some other issue that fixes since that time have resolved.
